subsys/caf/modules/ble_smp.c

```diff
@@ -55,10 +55,6 @@ static bool app_event_handler(const struct app_event_header *aeh)
 
 		if (check_state(event, MODULE_ID(main), MODULE_STATE_READY)) {
 			mgmt_callback_register(&mgmt_callback);
-			img_mgmt_register_group();
-#ifdef CONFIG_MCUMGR_CMD_OS_MGMT
-			os_mgmt_register_group();
```

Comment:
btw. Do we still need to explicitly call the `smp_bt_register` here? I think it should be done automatically now.

Reply:
Fixed.